app/src/main/java/com/simple/checkstyledemo/MainActivity.java

```diff
@@ -1,13 +1,21 @@
 package com.simple.checkstyledemo;
 
-import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.app.Application;
 
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        int i = 3;
+        test(i);
+    }
+
+    private void test(int a) {
+        if (a > 3) return;
```

Comment:
**Reporter**: PMD
**Rule**: IfStmtsMustUseBraces
**Severity**: WARN
**File**: app/src/main/java/com/simple/checkstyledemo/MainActivity.java L18

Braces https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/braces.html#IfStmtsMustUseBraces

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*